### DISCUSSION TITLE
Are there any limits to how many users can have a role different than Viewer? The option to invite the user as anything else than viewer is disabled

### DISCUSSION THREAD

**Discussion by klys-equinix** [graphql-hive/console#4271] [Help]
I added several users to my organization. When i want to invite more, i can only select the Viewer role:
<img width="552" alt="Screenshot 2024-03-18 at 13 35 37" src="https://github.com/kamilkisiela/graphql-hive/assets/20441031/9a0bbb43-27c6-4133-ba00-f767d8ad393e">

**Comment by klys-equinix** [comment]
@kamilkisiela  ?

**Comment by kamilkisiela** [comment]
https://github.com/kamilkisiela/graphql-hive/issues/4290
